src/app/features/tasks/task/task.component.mixins.scss

```diff
@@ -56,19 +56,6 @@ $task-c-drag-drop-bg-dark: $dark12;
 $task-c-selected-bg-light: $light-theme-selected-task-bg-color;
 $task-c-selected-bg-dark: $dark4;
 
-
-@mixin isHostFocus($direct-parent-only-sel: false) {
-  @if $direct-parent-only-sel {
-    :host-context(.isNoTouchOnly):focus > #{$direct-parent-only-sel} > & {
-      @content;
-    }
-  } @else {
-    :host-context(.isNoTouchOnly):focus & {
-      @content;
-    }
-  }
-}
```

Comment:
To my understanding this mixin gets called only once from task.component.scss. There an argument gets passed making the else statement unnecessary? In addition to that it seems to work perfectly with a small change in task.component.scss.

Reply:
Yes you're right! Good catch again! This is left-over code... Though I would prefer not mix in refactoring into this PR as it is complex enough already :)